packages/frontier-libraries/stdlib/README.md

```diff
@@ -1,10 +1,63 @@
+
+<div align="left">
+  <br/>
+  <a href="https://www.realdecoy.com/jamaica/" title="REALDECOY">
+    <img width=400px src="https://www.realdecoy.com/wp-content/uploads/2019/02/Realdecoy-logo-transparent.png" alt="rd logo">
+  </a>
+  <br/>
+</div>
+
 # @rdfrontier/stdlib
 
+### Frontier Standard Library
 A collection of functions/logic that are written in pure JavaScript with limited
-dependencies. These functions expose generic concepts.
+dependencies. These functions expose generic concepts available for use throughout the CLI. 
```

Comment:
available for use throughout the CLI. The standard library can be used on any javascript or typescript enabled project and not just CLI stuff